db/migrate/20190103113715_create_users.rb

```diff
@@ -0,0 +1,16 @@
+class CreateUsers < Mongoid::Migration
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
+      t.string :uid
+      t.string :organisation_slug
+      t.string :organisation_content_id
+      t.text :permissions
+      t.boolean :remotely_signed_out, :default => false
+      t.boolean :disabled, :default => false
+
+      t.timestamps null: false
+    end
```

Comment:
Do we need to do this for Mongoid? I thought in it's schemaless ways that it wouldn't need such things.

Reply:
ah probably right. removed